src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -107,3 +107,13 @@ admin_orders_change_currency:
     _controller: PrestaShopBundle:Admin/Sell/Order/Order:changeCurrency
   requirements:
     orderId: \d+
+
+admin_orders_resend_email:
+  path: /{orderId}/history/{orderHistoryId}/statuses/{orderStatusId}/resend-email
```

Comment:
@matks about this path?

Reply:
Makes sense 👍 from the path we can guess the target, so it works